src/api/xdsl/rma/xdsl-rma.v6.service.js

```diff
@@ -0,0 +1,48 @@
+angular.module('ovh-api-services').service('OvhApiXdslRMAV6', ($resource, $cacheFactory) => {
+  const cache = $cacheFactory('OvhApiXdslRMAV6');
+  const queryCache = $cacheFactory('OvhApiXdslRMAV6Query');
+
+  const interceptor = {
+    response(response) {
+      cache.remove(response.config.url);
+      queryCache.removeAll();
+      return response.resource;
+    },
+  };
+
+  const resource = $resource('/xdsl/:xdslId/rma', {
```

Comment:
Shouldn't be:

```suggestion
  const resource = $resource('/xdsl/:xdslId/rma/:id', {
```

instead?

Reply:
I need the list too

Reply:
That's the `query` action purpose.